### PR TITLE
fix indentation and closing braces

### PR DESCRIPTION
**Pull request by lamont-granquist**

### CODE REVIEW

**Comment by lamont-granquist** [comment]
easier to view this PR without the whitespace:

https://github.com/sswelm/KSP-Interstellar-Extended/pull/2/files?w=0

other than the few added lines, it removes a stray ending '}' in the middle of it (which the KSP config parser happily ignore) and adds a '}' on the end which was missing.

File: GameData/WarpPlugin/RealPlume/Quantum Vacuum.cfg [hunk 1]
```diff
@@ -1,170 +1,175 @@
...
-        	{
-            		%powerEffectName = Hydynelox-A7
-        	}
-    	}
-}
\ No newline at end of file
+                %powerEffectName = Hydynelox-A7
+        }
+
+        {
```
Comment (lamont-granquist) [inline GameData/WarpPlugin/RealPlume/Quantum Vacuum.cfg]: i think this does nothing and could probably be removed?

File: GameData/WarpPlugin/RealPlume/Quantum Vacuum.cfg [hunk 2]
```diff
@@ -1,170 +1,175 @@
...
+                %type = ModuleEnginesRF
+                @CONFIG[electric_argon]
+                {
+                        %powerEffectName = Hydynelox-A7
+                }
+                @CONFIG[electric_xenon]
+                {
+                        %powerEffectName = Hydynelox-A7
+                }
+                {
```
Comment (lamont-granquist) [inline GameData/WarpPlugin/RealPlume/Quantum Vacuum.cfg]: same with this?

**Comment by lamont-granquist** [comment]
the last section still looks kinda whack, not sure how it is supposed to be written...

**Comment by sswelm** [comment]
let me get this clear, you added a few newlines to improve readability?

**Comment by lamont-granquist** [comment]
no you have a stray extra closing curly brace on line 135, and you're missing one at the end of the file.

the top part looks like this:

```
@PART[InterstellarPlasmaThrusterUpgraded]:FOR[RealPlume]:NEEDS[SmokeScreen] //Quantum Vacuum Plasma Thruster
{
...stuff with matching braces...
}
}               <---- THIS GUY RIGHT HERE IS LONELY
```

then, after fixing the indentation for clarity the last bit current looks like:

```
@MODULE[ModuleEngineConfigs]
{        <--- ALSO LACKING A CLOSING BRACE
        %type = ModuleEnginesRF
        @CONFIG[electric_qvp]
        {
                %powerEffectName = Hydynelox-A7
        }
        @CONFIG[electric_red]
        {
                %powerEffectName = Hydynelox-A7
        }

        {
                %type = ModuleEnginesRF
                @CONFIG[electric_argon]
                {
                        %powerEffectName = Hydynelox-A7
                }
                @CONFIG[electric_xenon]
                {
                        %powerEffectName = Hydynelox-A7
                }
                {
                        %type = ModuleEnginesRF
                        @CONFIG[electric_hydrogen]
                        {
                                %powerEffectName = Hydynelox-A7
                        }
                        @CONFIG[electric_ammonia]
                        {
                                %powerEffectName = Hydynelox-A7
                        }
                }
        }
```

**Comment by lamont-granquist** [comment]
also i'm writing a config node parser and i'm genuinely just curious if there's any difference between that last section and this:


```
@MODULE[ModuleEngineConfigs]
{
        %type = ModuleEnginesRF
        @CONFIG[electric_qvp]
        {
                %powerEffectName = Hydynelox-A7
        }
        @CONFIG[electric_red]
        {
                %powerEffectName = Hydynelox-A7
        }
        @CONFIG[electric_argon]
        {
                %powerEffectName = Hydynelox-A7
        }
        @CONFIG[electric_xenon]
        {
                %powerEffectName = Hydynelox-A7
        }
        @CONFIG[electric_hydrogen]
        {
                %powerEffectName = Hydynelox-A7
        }
        @CONFIG[electric_ammonia]
        {
                %powerEffectName = Hydynelox-A7
        }
}
```

I don't think the extra braces without a leading node type / module manager definition are doing anything?  neither is the repeated `%type` statements?  and therefore it looks like a bunch of copy-paste errors here?  if its semantically any different that would help me sort out what is really going on in the KSP ConfigNode parser...

**Pull request merged**